scripts/src/java/org/oppia/android/scripts/maven/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+"""
+Libraries corresponding to attributing third-party library Copyright licenses in Oppia-android.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_library")
+
+kt_jvm_library(
+    name = "maven_dependencies_lib",
+    srcs = [
+        "maveninstall/MavenListDependencies.kt",
+        "maveninstall/MavenListDependency.kt",
+        "maveninstall/MavenListDependencyTree.kt",
+    ],
+    visibility = ["//visibility:public"],
```

Comment:
Does this need to be public?

Reply:
Fixed.

Reply:
Updated to `scripts:oppia_script_binary_visibility `